projects/plugins/jetpack/extensions/blocks/blog-stats/blog-stats.php

```diff
@@ -95,7 +95,7 @@ function load_assets( $attributes ) {
 		_n( 'hit', 'hits', $stats, 'jetpack' )
 	);
 
-	$label = $attributes['label'] ? $attributes['label'] : $fallback_label;
+	$label = $attributes['label'] ?? $fallback_label;
```

Comment:
This represents a behavior change if `$attributes['label']` is a non-null falsey value (e.g. empty string).

Probably the previous version should have looked like this instead
```suggestion
	$label = empty( $attributes['label'] ) ? $fallback_label : $attributes['label'];
```